server-spi-private/src/main/java/org/keycloak/models/utils/ModelToRepresentation.java

```diff
@@ -1018,10 +1018,15 @@ public static PermissionTicketRepresentation toRepresentation(PermissionTicket t
             representation.setResourceName(resource.getName());
             KeycloakSession keycloakSession = authorization.getKeycloakSession();
             RealmModel realm = authorization.getRealm();
-            UserModel owner = keycloakSession.users().getUserById(ticket.getOwner(), realm);
+            UserModel userOwner = keycloakSession.users().getUserById(ticket.getOwner(), realm);
             UserModel requester = keycloakSession.users().getUserById(ticket.getRequester(), realm);
             representation.setRequesterName(requester.getUsername());
-            representation.setOwnerName(owner.getUsername());
+            if (userOwner != null) {
+                representation.setOwnerName(userOwner.getUsername());
+            } else {
+                ClientModel clientOwner = keycloakSession.clientStorageManager().getClientById(ticket.getOwner(), realm);
```

Comment:
You should get the client from the `realm`. For instance, using `org.keycloak.models.RealmModel#getClientById`.